dll/cpl/powercfg/lang/ro-RO.rc

```diff
@@ -4,110 +4,110 @@
  * PURPOSE:     Romanian resource file
  * TRANSLATORS: Copyright 2009 Petru Dimitriu <petrimetri@gmail.com>
  *              Copyright 2011-2018 Ștefan Fulea <stefan.fulea@mail.com>
- *              Copyright 2022-2023 Andrei Miloiu <miloiuandrei@gmail.com>
+ *              Copyright 2022-2024 Andrei Miloiu <miloiuandrei@gmail.com>
  */
 
 LANGUAGE LANG_ROMANIAN, SUBLANG_NEUTRAL
 
 IDD_POWERSCHEMESPAGE_ACDC DIALOGEX 0, 0, 252, 220
 STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
-CAPTION "Regim de consum"
+CAPTION "Scheme de alimentare"
 FONT 8, "MS Shell Dlg"
 BEGIN
     ICON IDI_ACDC, IDI_ACDC, 7, 7, 0, 0
-    LTEXT "Alegeți un regim de consum al energiei cât mai potrivit pentru dumneavoastră. Aveți în vedere că modificările aduse vor fi memorate în regimul curent.", -1, 37, 6, 208, 36
-    GROUPBOX "&Regimuri de consum", -1, 6, 43, 240, 50
+    LTEXT "Selectaţi schema de alimentare cu cele mai potrivite setări pentru computer. Reţineţi că modificarea setărilor de mai jos va afecta schema selectată.", -1, 37, 6, 208, 36
+    GROUPBOX "&Scheme de alimentare", -1, 6, 43, 240, 50
     COMBOBOX IDC_ENERGYLIST, 14, 54, 224, 92, CBS_DROPDOWNLIST | CBS_AUTOHSCROLL | WS_VSCROLL | WS_TABSTOP
-    PUSHBUTTON "Salvea&ză…", IDC_SAVEAS_BTN, 85, 72, 80, 14, WS_DISABLED
-    PUSHBUTTON "Șt&erge", IDC_DELETE_BTN, 173, 72, 64, 14, WS_DISABLED
+    PUSHBUTTON "Sal&vare ca…", IDC_SAVEAS_BTN, 85, 72, 80, 14, WS_DISABLED
+    PUSHBUTTON "Şt&ergere", IDC_DELETE_BTN, 173, 72, 64, 14, WS_DISABLED
     GROUPBOX "", IDC_GRPDETAIL, 6, 95, 240, 119
-    LTEXT "Când calculatorul este:", IDC_SAT, 12, 110, 60, 15
-    LTEXT "În priză", IDC_SAC, 100, 110, 50, 15
+    LTEXT "Când computerul este alimentat:", IDC_SAT, 12, 110, 60, 15
+    LTEXT "De la reţeaua electrică", IDC_SAC, 100, 110, 50, 15
     ICON IDI_AC, IDC_IAC, 141, 122, 0, 0
-    LTEXT "Pe baterii", IDC_SDC, 173, 110, 50, 15
+    LTEXT "De la baterii", IDC_SDC, 173, 110, 50, 15
     ICON IDI_DC, IDC_IDC, 218, 122, 0, 0
-    LTEXT "Închiderea e&cranului:", IDC_MONITOR, 13, 148, 85, 10
+    LTEXT "Stingere &monitor:", IDC_MONITOR, 13, 148, 85, 10
     COMBOBOX IDC_MONITORACLIST, 100, 145, 65, 150, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
     COMBOBOX IDC_MONITORDCLIST, 173, 145, 65, 150, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
-    LTEXT "Oprirea &discurilor:", IDC_DISK, 13, 163, 85, 10
+    LTEXT "Oprire d&iscuri fixe:", IDC_DISK, 13, 163, 85, 10
```

Comment:
MS really used an i as accelerator? That is untypically. Please attach a pic from the romanian MS dlg for proof.

Reply:
Yes, it's i letter an accelerator. Here is the proof.

![2024-04-14 11_31_47-Resource Hacker - powercfg cpl mui](https://github.com/reactos/reactos/assets/113553683/dcf1f189-2ccd-4bce-b371-fc8544a965b6)

Reply:
Resolved, thanks.